.github/workflows/static-analysis.yml

```diff
@@ -7,39 +7,39 @@ on:
 env:
   CC: clang-9
   CXX: clang++-9
-  ADDITIONAL_CMAKE_FLAGS:
+  ADDITIONAL_CMAKE_FLAGS: ""
 
 jobs:
   StyleAndFormatting:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - run: git submodule update --init
-    - name: Validation
-      run: tools/ci/staticValidation.sh "$GITHUB_WORKSPACE"
-    - name: Formatting
-      uses: Flamefire/clang-format-lint-action@master
-      with:
-        source: 'extras libs tests external/libendian external/liblobby external/libsiedler2 external/libutil external/mygettext external/s25edit external/s25update'
-        clangFormatVersion: 9
-    - name: Lint markdown files
-      uses: avto-dev/markdown-lint@v1.3.0
-      with:
-        args:  --ignore data/RTTR/MAPS .
-        ignore: external
+      - uses: actions/checkout@v2
+      - run: git submodule update --init
+      - name: Validation
+        run: tools/ci/staticValidation.sh "$GITHUB_WORKSPACE"
+      - name: Formatting
+        uses: DoozyX/clang-format-lint-action@v0.9
+        with:
+          source: "extras libs tests external/libendian external/liblobby external/libsiedler2 external/libutil external/mygettext external/s25edit external/s25update"
+          clangFormatVersion: 10
+      - name: Lint markdown files
+        uses: avto-dev/markdown-lint@v1.3.0
+        with:
+          args: --ignore data/RTTR/MAPS .
+          ignore: external
 
   Clang-Tidy:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - run: git submodule update --init
-    - name: Install dependencies
-      run: |
-        sudo apt-get update && sudo apt-get install -y \
-          clang-9 clang-tidy-9 \
-          libsdl2-dev libsdl2-mixer-dev libcurl4-openssl-dev libbz2-dev libminiupnpc-dev liblua5.2-dev gettext \
-          libboost-filesystem-dev libboost-program-options-dev libboost-thread-dev libboost-test-dev libboost-locale-dev libboost-iostreams-dev
-    - name: Clang-Tidy Main
-      run: tools/ci/runClangTidy.sh main
-    - name: Clang-Tidy External
-      run: tools/ci/runClangTidy.sh external
+      - uses: actions/checkout@v2
+      - run: git submodule update --init
+      - name: Install dependencies
+        run: |
+          sudo apt-get update && sudo apt-get install -y \
+            clang-9 clang-tidy-9 \
```

Comment:
shouldn't it install clang 10?

Reply:
Not yet. clang and clang-tidy must match and clang-tidy-10 contains checks that I want to enable in the near future but which would fail now. Note that clang-format-10 is not dependent on clang-10